src/main/java/racingcar/model/RoundNumber.java

```diff
@@ -0,0 +1,30 @@
+package racingcar.model;
+
+import racingcar.constant.ExceptionMessage;
+
+public class RoundNumber {
+    private int number;
+
+    public RoundNumber(String roundString) {
+        parseInt(roundString);
+        validatePositiveNumber();
+    }
```

Comment:
저 같은 경우에는 parse의 결과에 대해서 validate함을 쉽게 파악할 수 있도록
```suggestion
    public RoundNumber(String roundString) {
        Integer number = parseInt(roundString);
        validatePositiveNumber(number);
        this.number = number;
    }
```
와 같이 구성될 수 있도록 할 것 같아요!

Reply:
말씀 듣고 보니 제 코드가 어딘가 마음에 안들었던 이유를 알 것 같네요 ㅎㅎㅎ 
좋은 의견 감사합니다!!